packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -0,0 +1,212 @@
+          remoteAddress: `0x${string}`,
+          amountValue: bigint,
+          status: 'confirmed' | 'failed' | 'pending',
+        ) {
+          const key =
+            `${chainId}:${remoteAddress}:${amountValue}` as CCTPTransactionKey;
+
+          if (!cctpTransactionRegistry.has(key)) {
+            trace('No pending CCTP transaction found for key:', key);
+            trace('Available keys in registry:', [
+              ...cctpTransactionRegistry.keys(),
+            ]);
+            throw assert.error(
+              X`No pending CCTP transaction found matching provided details: ${q(key)}`,
+            );
+          }
+
+          const registryEntry = cctpTransactionRegistry.get(key);
+
+          switch (status) {
+            case 'confirmed':
+              trace(`CCTP transaction confirmed - resolving pending operation`);
+              registryEntry.vowKit.resolver.resolve();
+              cctpTransactionRegistry.delete(key);
+              return;
+
+            case 'failed':
+              trace(`CCTP transaction failed - rejecting pending operation`);
+              registryEntry.vowKit.resolver.reject(
+                assert.error('CCTP transaction failed'),
+              );
+              cctpTransactionRegistry.delete(key);
+              return;
+          }
+        },
+      },
+      invitationMakers: {
+        SettleCCTPTransaction() {
+          trace('SettleCCTPTransaction');
+          const { service } = this.facets;
+
+          const proposalShape = M.splitRecord(
+            { give: {}, want: {} },
+            { exit: M.any() },
+            {},
+          );
+
+          const settlementHandler = async (
+            seat: ZCFSeat,
+            offerArgs: CCTPSettlementOfferArgs,
+          ) => {
+            mustMatch(offerArgs, ResolverOfferArgsShapes.SettleCCTPTransaction);
+            const { txDetails, remoteAxelarChain } = offerArgs;
+
+            trace('CCTP transaction settlement:', {
+              amount: txDetails.amount,
+              remoteAddress: txDetails.remoteAddress,
+              status: txDetails.status,
+              remoteAxelarChain,
+            });
+
+            seat.exit();
+
+            try {
+              service.settleCCTPTransaction(
+                remoteAxelarChain,
+                txDetails.remoteAddress,
+                txDetails.amount,
+                txDetails.status,
+              );
+
+              return 'CCTP transaction settlement processed';
+            } catch (error) {
+              return `CCTP transaction settlement failed: ${error.message || 'Unknown error'}`;
+            }
+          };
+
+          return zcf.makeInvitation(
+            settlementHandler,
+            'settleCCTPTransaction',
+            undefined,
+            proposalShape,
+          );
+        },
+      },
+    },
+  );
+};
+export type ResolverInvitationMakers = {
+  SettleCCTPTransaction: () => Promise<Invitation<string>>;
+};
```

Comment:
this has the same IDE navigation issue.

```suggestion
export type ResolverInvitationMakers = ResolverKit['invitationMakers'];
```